omisego-sdk/src/main/java/co/omisego/omisego/custom/retrofit2/adapter/OMGCall.kt

```diff
@@ -7,12 +7,12 @@ package co.omisego.omisego.custom.retrofit2.adapter
  * Copyright © 2017-2018 OmiseGO. All rights reserved.
  */
 
-import co.omisego.omisego.custom.Callback
+import co.omisego.omisego.custom.OMGCallback
 import retrofit2.Response
 
 interface OMGCall<T> {
     fun cancel()
     fun clone(): OMGCall<T>
     fun execute(): Response<T>
```

Comment:
Shouldn't the response be changed to `OMGResponse<T>` now to match the changes?

Reply:
Done